transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/CamelConfigurationHelper.java

```diff
@@ -10,39 +10,32 @@
 package org.jboss.tools.fuse.transformation.editor.internal.util;
 
 import java.io.File;
-
 import org.eclipse.core.runtime.NullProgressMonitor;
 import org.fusesource.ide.camel.model.service.core.io.CamelIOHandler;
 import org.fusesource.ide.camel.model.service.core.model.CamelFile;
 import org.jboss.tools.fuse.transformation.core.camel.CamelConfigBuilder;
 
 public final class CamelConfigurationHelper {
 
-    private CamelFile camelModel;
-    private CamelConfigBuilder configBuilder;
-    private CamelIOHandler handler;
-    
-    private CamelConfigurationHelper(CamelFile camelModel, CamelIOHandler handler) {
-    	this.camelModel = camelModel;
-    	this.handler = handler;
+    public static CamelConfigurationHelper load(File contextFile) {
```

Comment:
I used to avoid to mix static and non-static so not sure.
SonarLint reporting it as a (little) issue but it might be a bad convention that they are pushing.